website/docs/sql_ddl.md

```diff
@@ -383,18 +383,57 @@ CREATE CATALOG hoodie_catalog
 The following is an example of creating a Flink table. Read the [Flink Quick Start](/docs/flink-quick-start-guide) guide for more examples.
 
 ```sql 
-CREATE TABLE hudi_table2(
+CREATE TABLE hudi_table(
   id int, 
   name string, 
   price double
 )
 WITH (
 'connector' = 'hudi',
-'path' = 's3://bucket-name/hudi/',
+'path' = 'file:///tmp/hudi_table',
 'table.type' = 'MERGE_ON_READ' -- this creates a MERGE_ON_READ table, default is COPY_ON_WRITE
 );
 ```
 
+### Create partitioned table
+
+The following is an example of creating a Flink partitioned table.
+
+```sql 
+CREATE TABLE hudi_table(
+  id BIGINT,
+  name STRING,
+  dt STRING,
+  hh STRING
+)
+PARTITIONED BY (`dt`)
+WITH (
+'connector' = 'hudi',
+'path' = 'file:///tmp/hudi_table',
+'table.type' = 'MERGE_ON_READ'
+);
+```
+
+### Create table with record keys and ordering fields
```

Comment:
Also consider adding a section on how to add Hudi specific configs to Flink similar to 'Setting Hudi configs' section in this page under Spark SQL.